### PR TITLE
Feature/use logos layout for companies using kedro

### LINKED ISSUE #58: Move to a logo-driven showcase of companies using Kedro
## Introduction

Request from Debanjan Banerjee: Can we make the clientele tab more on the face like other products like Databricks do?

image

I was showing this to a client lead DS and the current roulette style design is a bit difficult to read? The user should be able to read the whole list at once.

### PR DESCRIPTION
**Pull request by Huongg**
## Description
Fixes #58 

## Development notes

I've updated the new layout designed by @Mackay031 (with the screenshot below)

Screenshot 2022-09-21 at 12 42 54

## QA notes

it has been tested manually by me to ensure all the logos are in the right order and position (with design guidelines as below)
logos in place

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Added tests to cover my changes, where applicable

### CODE REVIEW

**Review by cvetankanechevska — APPROVED** [review]
Looks good to me! 🎉

**Comment by Huongg** [comment]
thank you @cvetankanechevska  @rashidakanchwala, the mobile and ipad versions are ready for review now too

File: components/companies-using-kedro/companies-using-kedro.module.scss [hunk 1]
```diff
@@ -17,100 +17,74 @@
...
 
   @include breakpoint(small) {
-    justify-content: left;
-    width: 100%;
+    margin-bottom: 2.5rem;
+    padding: 0;
   }
 }
 
 .right {
```
Comment (rashidakanchwala) [inline components/companies-using-kedro/companies-using-kedro.module.scss:41]: maybe we can remove this and incorporate styling in the other div.inner?

**Review by rashidakanchwala — APPROVED** [review]
LGTM. just a couple of minor comments on the old code

File: components/companies-using-kedro/companies-using-kedro.module.scss (hunk 1 above)
Comment (Huongg) [reply components/companies-using-kedro/companies-using-kedro.module.scss]: hey good spot, I just pushed my changes to remove them now

**Review by tynandebold — APPROVED** [review]
Great job, Huong! Thank you for this.

I'm approving with one question: shouldn't we still have a title for this section like we do for all of the others? To me it isn't clear what you're looking at when you're scrolling down the page and then just hit this matrix of logos. 

Another thing: the logos feel really squished together to me. We have so much air and space in the other modules but not in this one.

**Comment by Mackay031** [comment]
Hey, just responding. These logos are still part of case studies as they were before. I agree that could have the same background colour as Case Studies to visually connect it. We could swap out the bg colour of this and testimonials?

On spacing, I feel the desktop version is ok but we looking at adding some vertical spacing to the mobile version so allow for selection with fat thumbs....

**Comment by tynandebold** [comment]
> Hey, just responding. These logos are still part of case studies as they were before. I agree that could have the same background colour as Case Studies to visually connect it. We could swap out the bg colour of this and testimonials?
> 
> On spacing, I feel the desktop version is ok but we looking at adding some vertical spacing to the mobile version so allow for selection with fat thumbs....

Oh I see. Didn't realize it was still part of the **Case Studies** section since before it was **Used and loved by**.

**Comment by tynandebold** [comment]
`Our Community` sounds nice.

**Comment by Mackay031** [comment]
Do you perhaps know what the issue was with "Case Studies'? I think my reservation for changing it is the expectation of what happens when you click the logos. Even as it stands, we might even do well to have a line of copy about selecting the logos to see how Kedro is being used. If we put ourselves in the mind of the user:
- Do we know that we can click/tap on the logos? 
- If we see logo grids on websites, are they usually clickable or are they just to show where the product is used?
- What do we expect to happen if we do indeed know that we can click a logo? 
- A new window opens with links to different types of content (eg LinkedIn, Medium, Github). Collectively, how would we describe this content if not as Case Studies. 'Our community' does describe this group, but perhaps not the linked content. Does Case Studies feel to business? What about 'Kedro in use' or even something more fun "Kedro in the wild'

On another note, as per previous discussion, we should replace 'Telkomsel' and 'Beamery' in the grid as they appear above. 

Just my 2cents (this will be a £1 soon...)

Just adding @yetudada to this discussion.

**Comment by yetudada** [comment]
@Huongg this is looking great so far! I've just gone through all of the comments and I have these suggestions: 

- I don't think this section should be combined with "Case Studies". The more common pattern is to call this "Trusted By" (example [1](https://flyte.org/) and [2](https://greatexpectations.io/)) or even "Organizations using Kedro" (example [3](https://mlflow.org/)) but, equally, I would also be happy with "Our Community". The section definitely needs a title though. 
- Some of the points @Mackay031 raised could be addressed by updating all URLs to point to the company's website. Once again, this seems like a very, very common pattern. It would also allow this section to remain separate from "Case Studies" so that the content is not in two sections. 
- We still haven't solved outlying issues telling users they can click on the logos but, I imagine solving for this will be less important if it's just the company's URL.

**Comment by Huongg** [comment]
thank you @yetudada and @Mackay031. Below is the screenshots of what it looks like if we include either **"Our Community"** or the old version **"Used and loved by"**. I think if we go with **"Our Community"** or other similar suggestions, I agree that we should update the URL to just companies' websites/links, but if we decide to keep **"Used and loved by"**, potentially we can just keep the current URLs with case studies? 

Screenshot 2022-09-27 at 12 11 41


Screenshot 2022-09-27 at 13 07 59

**Comment by Mackay031** [comment]
Hey all,

- Agree that if the content post click is just a website, then it should **not** be included with 'Case Studies'. The content we currently link to feels like case studies.
- 'our community' feels more personal for me.
- Also agree, that if we are just linking to a website, then no need for any other copy above logos. In fact, i'd argue thats its better that they don't link out so that we potentially lose them.

**Comment by Huongg** [comment]
The latest changes I made from our discussion in standup this morning
- remove the link from the logo (@Mackay031 since we don't have link anymore, I removed the extra gap so the user can click on the link in the mobile version) 
- Make the "Our Community" font size the same as other header.

File: components/companies-using-kedro/companies-using-kedro.module.scss [hunk 2]
```diff
@@ -17,100 +17,54 @@
...
-    transparent,
-    transparent,
-    $color-dark
-  );
-  height: 30rem;
-  pointer-events: none;
-  position: relative;
+.title {
+  margin-bottom: 3rem;
+  text-align: center;
```
Comment (tynandebold) [inline components/companies-using-kedro/companies-using-kedro.module.scss:36]: We have a mixin that you can use here to make life easier. The font is now too small on mobile compared to the other section headers. [Reference link here](https://github.com/kedro-org/kedro-website/blob/main/components/features/features.module.scss#L24).

Comment (Huongg) [reply components/companies-using-kedro/companies-using-kedro.module.scss:37]: ah great thanks for letting me know 👍

**Pull request merged**